Gopkg.toml

```diff
@@ -0,0 +1,66 @@
+# Gopkg.toml example
+#
+# Refer to https://golang.github.io/dep/docs/Gopkg.toml.html
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+
+[[constraint]]
+  name = "github.com/gorilla/mux"
+  version = "~1.6.2"
+
+[[constraint]]
+  name = "github.com/jinzhu/gorm"
+  version = "~1.9.1"
+
+[[constraint]]
+  name = "github.com/prometheus/client_golang"
+  version = "0.9.1"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/vburenin/nsync"
+
+[[constraint]]
+  name = "gopkg.in/gormigrate.v1"
+  version = "~1.2.1"
+
+[[constraint]]
+  name = "k8s.io/client-go"
+  version = "10.0.0"
```

Comment:
You need to do some extra mojo with this to ensure that the right api-machinery is installed.  https://medium.com/programming-kubernetes/using-gos-dep-to-organize-your-kubernetes-client-go-dependencies-509ddc766ed3 is a good reference.

Reply:
Once I get this in, I'll ask Roman to also take a good look at this before merging.